daemon/network.go

```diff
@@ -338,11 +339,12 @@ func (daemon *Daemon) GetNetworkDriverList() map[string]bool {
 	networks := c.Networks()
 
 	for _, network := range networks {
-		driver := network.Type()
-		pluginList[driver] = true
+		pluginList = append(pluginList, network.Type())
```

Comment:
Good point, I assumed that can't happen from libnetwork, happy to handle it if that's the case